CalculatorApp.js

```diff
@@ -0,0 +1,69 @@
+// JavaScript source code
+var storage = 0;
+var display = "";
+var resultelement;
+
+function initializeGloabls() {
```

Comment:
initializeGloabls method is spelled incorrectly.

Reply:
corrected the method name